surfstat/python/matlab_functions.py

```diff
@@ -93,4 +72,106 @@ def colon(start,stop,increment=1):
         return r
     elif start == stop or r[-1] + increment == stop:
         r = np.append(r,stop)
-    return r
\ No newline at end of file
+    return r
+
+def accum(accmap, a, func=None, size=None, fill_value=0, dtype=None):
```

Comment:
There are bugs in the accum function; I'd remove it entirely and replace all occurrences of it with np.bincount

Reply:
Cool! 
I haven't come across with any bugs in accum. However, trusting on your experience with accum, it is now replaced with np.bincount.